src/jupyterlabcontroller/models/v1/event.py

```diff
@@ -26,7 +26,7 @@ class Event(BaseModel):
     type: EventType = Field(..., title="Type of the event")
     message: str = Field(..., title="Event message")
     progress: Optional[int] = Field(
-        None, title="Progress percentage", lt=100, gt=0
```

Comment:
Did we ever really define whether each endpoint was open or closed?

Reply:
I think it's closed, but JupyterHub should get 100% for itself for spawning since it has to wait for the lab service to start. It doesn't really matter for deletion, since nothing looks at those events except us. Using 100% for finishing the deletion felt right.